src/components/edge-auth0-sync.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <v-dialog
+    id="notification-dialog"
+    persistent
+    v-model="showEdgeSyncModal"
+    max-width="550"
+  >
+    <v-card>
+      <div class="container">
+        <div
+          class="flex"
+          style="justify-content: space-between;"
+        >
+          <div class="align-center">
+            <h3 style="font-weight: normal;">
+              Ambianic Edge Device Sync
+            </h3>
+          </div>
+
+          <div
+            class="icon"
+            @click="handleClose('DENIED')"
+          >
+            <v-icon
+              style="margin: 0.5rem 0;"
+              center
+              id="close-icon"
+              size="25"
+            >
+              mdi-close
+            </v-icon>
+          </div>
+        </div>
+
+        <hr>
+
+        <div>
+          <div v-if="EDGE_SYNC_COMPATIBILITY_STATUS === 'CHECKING'">
+            <br>
+            <v-progress-circular
+              indeterminate
+              :width="3.5"
+              id="compatibile-spinner"
+              :size="40"
+              color="primary"
+            />
+            <br>
+            <br>
+            <p id="checker-text">
+              Checking if your connected Edge Device is compatible with Ambianic Premium Services.
+            </p>
+            <br>
+          </div>
+
+          <v-list v-if="EDGE_SYNC_COMPATIBILITY_STATUS === 'OUTDATED'">
+            <br>
+            <div class="align-center">
+              <v-icon
+                style="margin: 0;"
+                center
+                id="outdated-icon"
+                size="45"
+              >
+                mdi-alert-outline
+              </v-icon>
+            </div>
+            <br>
+            <p id="outdated-text">
+              Your Edge Device is currently running on version {{ edgeVersion }} and is missing the new <a href="https://docs.ambianic.ai/users/premium-services/">Premium Services</a> feature.
+            </p>
+            <br>
+
+            <hr>
+            <div
+              class="flex"
+              style="margin: .5rem 0 0 0"
+            >
+              <div class="align-center">
+                <p style="margin: 0 .3rem; font-size: .95rem">
+                  Please restart your Ambianic Edge device to update your Ambianic Edge Installation.
```

Comment:
@vickywane restart is not required for ambianic edge to pull an update. docker watchtower does that automatically.
A more appropriate message might be:
"Your Ambianic Edge device is running an outdated software version. It should automatically update to the latest version within 24 hours. Please try again later or contact support if the issue persists."